tests/ui/author/if.stdout

```diff
@@ -1,32 +1,48 @@
 if_chain! {
     if let StmtKind::Local(ref local) = stmt.kind;
     if let Some(ref init) = local.init;
-    if let ExprKind::If(ref cond, ref then, Some(ref else_)) = init.kind;
-    if let ExprKind::Block(ref block) = else_.kind;
+    if let Some(If { cond @ cond, then @ then, r#else: Some(else_expr)}) = higher::If::hir(init.kind)
+    if let ExprKind::Lit(ref lit) = cond.kind;
+    if let LitKind::Bool(true) = lit.node;
+    if let ExprKind::Block(ref block, ref label) = then.kind;
     if block.stmts.len() == 1;
     if let StmtKind::Semi(ref e, _) = block.stmts[0].kind
     if let ExprKind::Binary(ref op, ref left, ref right) = e.kind;
     if BinOpKind::Eq == op.node;
-    if let ExprKind::Lit(ref lit) = left.kind;
-    if let LitKind::Int(2, _) = lit.node;
-    if let ExprKind::Lit(ref lit1) = right.kind;
-    if let LitKind::Int(2, _) = lit1.node;
+    if let ExprKind::Lit(ref lit1) = left.kind;
+    if let LitKind::Int(1, LitIntType::Unsuffixed) = lit1.node;
```

Comment:
Hmm how did the number change?

```suggestion
    if let LitKind::Int(2, LitIntType::Unsuffixed) = lit1.node;
```

Reply:
Looks like it used to check the else block first. https://github.com/rust-lang/rust-clippy/blob/310ecb0a5c0d9161679c256052822f34c6eb548a/tests/ui/author/if.stdout#L4-L5